pkg/chromium/browser.go

```diff
@@ -69,15 +83,80 @@ type RenderingOptions struct {
 
 	// Timeout is the duration of time we'll wait on render requests to complete.
 	Timeout time.Duration
+	// TimeBetweenScrolls is how long to wait on a page of the rendered website before scrolling further.
+	TimeBetweenScrolls time.Duration
 }
 
-// RenderPDF visits the website and waits for it to be fully rendered.
+func (r *RenderingOptions) SetHeader(key, value string) {
+	if r.Headers == nil {
+		r.Headers = make(map[string]string)
+	}
+	r.Headers[key] = value
+}
+
+func (r *RenderingOptions) Normalise(prototype RenderingOptions) error {
+	if strings.HasPrefix(r.URL, "socket://") {
```

Comment:
Do we want to support it at some point?

Reply:
Probably not. Again, TS does this, so I just copied the behaviour.